Okta/OktaOidc/OktaOidc.swift

```diff
@@ -25,16 +25,21 @@ public class OktaOidc: NSObject {
 
     @objc public func authenticate(withSessionToken sessionToken: String,
                                    callback: @escaping ((OktaOidcStateManager?, Error?) -> Void)) {
-        OktaOidcAuthenticateTask(config: configuration, oktaAPI: OktaOidcRestApi())
-        .authenticateWithSessionToken(sessionToken: sessionToken) { authState, error in
+        OktaOidcAuthenticateTask(config: configuration,
+                                 oktaAPI: OktaOidcRestApi(delegate: configuration.requestCustomizationDelegate)).authenticateWithSessionToken(sessionToken: sessionToken,
+                                                                                                                          delegate: configuration.requestCustomizationDelegate,
+                                                                                                            callback: { (authState, error) in
             guard let authState = authState else {
                 callback(nil, error)
                 return
             }
             
             let authStateManager = OktaOidcStateManager(authState: authState)
+            if let delegate = self.configuration.requestCustomizationDelegate {
+                authStateManager.restAPI = OktaOidcRestApi(delegate: delegate)
```

Comment:
Please remove `delegate` from `OktaOidcRestApi` constructor and use property injection

Reply:
I've removed delegate from init, thanks.